protected/models/ar/SurveyResponse.php

```diff
@@ -54,6 +56,109 @@ public static function labels(): array
         );
     }
 
+    public function getAccessibility(): string
+    {
+        // TODO: Implement getAccessibility() method.
```

Comment:
Remove all these.

Reply:
I left these on purpose since I did not look into the proper implementation of these functions, I did remove the comments on the ones that I did implement. Thought they can be picked up when implementing the dashboarding for the HF layer.